jupyter-docker/custom/jupyter_localize_extension.py

```diff
@@ -1,6 +1,7 @@
 import distutils.util
```

Comment:
Seems like there may be other py code changes needed to this file. See: https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-fiab-test-runner/6563/artifact/output/gpalloc-qa-master-4savc2m-automation-test-ajk4iltzz-delete-jupyter.log

```
[E 16:10:21.573 NotebookApp] Uncaught exception POST /notebooks/gpalloc-qa-master-4savc2m/automation-test-ajk4iltzz/api/localize (127.0.0.1)
    HTTPServerRequest(protocol='http', host='127.0.0.1:8000', method='POST', uri='/notebooks/gpalloc-qa-master-4savc2m/automation-test-ajk4iltzz/api/localize', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Host': '127.0.0.1:8000', 'Authorization': 'Bearer ya29.GoABSAZbQVjdc3rGz77wcOzGBrDBF_HlpAzYfMctoOTMuOxaNX6cTGugvK74ygMJVCo00lNV2PWFAsMmU2mYNjYILbcMSiGGA3fbzSSQ4D8LvAnODeqJYYqQds7uXAkiPGCSMX7Oox8PjgsDUn-6XOKlLZHCi7Pffaz7U8q866YpmiU', 'Cookie': 'LeoToken=ya29.GoABSAZbQVjdc3rGz77wcOzGBrDBF_HlpAzYfMctoOTMuOxaNX6cTGugvK74ygMJVCo00lNV2PWFAsMmU2mYNjYILbcMSiGGA3fbzSSQ4D8LvAnODeqJYYqQds7uXAkiPGCSMX7Oox8PjgsDUn-6XOKlLZHCi7Pffaz7U8q866YpmiU', 'User-Agent': 'akka-http/10.1.2', 'X-Forwarded-For': '35.224.147.226, 35.225.42.147', 'X-Forwarded-Host': 'leonardo-fiab.dsde-qa.broadinstitute.org, app:8080', 'X-Forwarded-Server': 'leonardo-fiab.dsde-qa.broadinstitute.org, automation-test-ajk4iltzz-m.c.gpalloc-qa-master-4savc2m.internal', 'Content-Type': 'application/json', 'Connection': 'Keep-Alive', 'Content-Length': '254'})
    Traceback (most recent call last):
      File "/etc/jupyter/custom/jupyter_localize_extension.py", line 59, in _localize_data_uri
        destout.write(uri.data)
    TypeError: write() argument must be str, not bytes
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/usr/local/lib/python3.6/dist-packages/tornado/web.py", line 1510, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/etc/jupyter/custom/jupyter_localize_extension.py", line 129, in post
        failures = tornado.ioloop.IOLoop().run_sync(lambda: self.localize(pathdict))
      File "/usr/local/lib/python3.6/dist-packages/tornado/ioloop.py", line 458, in run_sync
        return future_cell[0].result()
      File "/usr/local/lib/python3.6/dist-packages/tornado/concurrent.py", line 238, in result
        raise_exc_info(self._exc_info)
      File "<string>", line 4, in raise_exc_info
      File "/usr/local/lib/python3.6/dist-packages/tornado/gen.py", line 292, in wrapper
        result = func(*args, **kwargs)
      File "/usr/lib/python3.6/types.py", line 248, in wrapped
        coro = func(*args, **kwargs)
      File "/etc/jupyter/custom/jupyter_localize_extension.py", line 90, in localize
        success = self._localize_data_uri(locout, source, dest)
      File "/etc/jupyter/custom/jupyter_localize_extension.py", line 65, in _localize_data_uri
        locout.write('{}: unexpected error: {}\n'.format(sys.exc_info()[0]))
    IndexError: tuple index out of range
```